assets/custom-icons/_data/custom-icons.json

```diff
@@ -185,6 +188,9 @@
     {
       "title": "Mozilla"
     },
+    {
+      "title": "MyFRITZ!Net"
```

Comment:
Since the file name is different from the title, could you add a `slug` that has the file name (`myfritz`)?